_includes/get_a_quote.html

```diff
@@ -8,180 +8,287 @@ <h1>Get a quote</h1>
         We are experts in working in the most efficient way possible, playing to each others' strengths, making sure you get the most value delivered for your money.
     </p>
     <p>
-        When you hire us, we will have a trial period of a month, after which we will evaluate our working relationship, to ensure that you are happy with the work we are doing for you.
+        If you get in touch, we will first arrange a meeting to talk about your concrete business situation.
+        We will suggest a small project for you to get familiar with how we work, and then hopefully initiate a longer, sustainable relationship, where we help you improve your business's software.
     </p>
     <p>
-        If you want to hire us, see if we can make something happen within your budget here:
+        Tell us a bit about your business and why you are considering getting help building software. Get in touch if you want to hear from us:
```

Comment:
I think the form is quite self-explanatory and I think that the text risk sounding like we require that the potential costumer needs to be too clarified about what they want. So maybe we don't need this paragraph at all, or maybe make it shorter and more open:

If you got any questions, got an idea or a problem or simply want to get in touch, then don't hesitate to contact us: